gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadPlugin.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Yahoo Inc. 2016, see https://github.com/flurry/upload-clients/blob/master/LICENSE.txt for full details
+ */
+package com.flurry.android.symbols
+
+import com.android.build.gradle.api.BaseVariant
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import com.flurry.proguard.UploadProGuardMapping
+
+/**
+ * A Gradle plugin that finds the generated ProGuard file and sends it to Flurry's crash service
+ */
+class SymbolUploadPlugin implements Plugin<Project> {
+    public static final String CONFIGURATION_KEY = "flurryCrash"
+    public static final String API_KEY = "api-key"
+    public static final String TOKEN = "token"
+    public static final String FLURRY_UUID_KEY = "com.flurry.crash.map_id"
+
+    @Override
+    void apply(Project project) {
+        UploadProGuardMapping.logger = project.logger
+        UploadProGuardMapping.failHard = false
```

Comment:
On second thought, this should be the default behavior and the CLI should set `failHard` to `true`